style.css

```diff
@@ -23,10 +23,6 @@ html, body {
     display: none;
 }
 
-.visible {
```

Comment:
I like that you changed the class name for the light box so that it is more specific.